CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 
+### 1.0.2 (2018-11-13):
+
+* Use `shim.getDatabaseNameFromUseQuery()` instead of deep-linking into agent.
```

Comment:
Might want to go with "Used" for consistent tense. Or something like "Replaced deep-link into agent to retrieve database name with call to `shim.getDatabaseNameFromUseQuery()`.

Reply:
Done!